proftpd/lang/fr

```diff
@@ -420,7 +447,10 @@ log_anonm=FTP anonyme édité manuellement pour le serveur  $1
 log_anon=$1 modifié dans le FTP anonyme pour le serveur $2
 log_start=proftpd démarré
 log_apply=modifications appliquées
+
 eafter=Vérification de la configuration échouée : Modifications de $1 non sauvegardées.
 proftpd_apply=Appliquer les modifications
 proftpd_stop=Stop ProFTPd
 proftpd_start=Démarrer ProFTPd
+
+__norefs=1
```

Comment:
This is not needed for any language but English.

No worries, I will run language manager script on it, it will fix it, and automatically remove _.auto_ files/strings that were already translated.